staging_vespalib/src/vespa/vespalib/util/clock.h

```diff
@@ -20,7 +20,7 @@ class Clock : public FastOS_Runnable
     Clock(const Clock &);
     Clock & operator = (const Clock &);
 
-    mutable fastos::SteadyTimeStamp _timeNS;
+    mutable volatile int64_t  _timeNS;
```

Comment:
actually, to ensure it is visible to other threads, it needs to be atomic

Reply:
Fixed, memory_order_relaxed.